config/fields/structure.php

```diff
@@ -91,18 +91,24 @@
         }
     ],
     'computed' => [
-        'default' => function () {
-            return $this->rows($this->default);
-        },
-        'value' => function () {
-            return $this->rows($this->value);
+        'form' => function () {
+            return new Form([
+                'fields' => $this->fields,
+                'model'  => $this->model
+            ]);
```

Comment:
I have an idea about failing tests. It can give you an idea. Because it is fixed when I move it from `computed` to `methods`.

I guess calling `$form->fill()` not affects to computed `$this->form` variable.

Reply:
Ah, that makes a lot of sense!